src/core/MOM_CoriolisAdv.F90

```diff
@@ -230,6 +232,10 @@ subroutine CorAdCalc(u, v, h, uh, vh, CAu, CAv, OBC, AD, G, GV, US, CS)
   ! 3D diagnostics hf_gKEu etc. are commented because there is no clarity on proper remapping grid option.
   ! The code is retained for degugging purposes in the future.
 
+! Diagnostics for thickness multiplied momentum budget terms
+  real, allocatable, dimension(:,:,:) :: h_gKEu, h_rvxv ! [L2 T-2 ~> m2 s-2].
```

Comment:
The documented units of these diagnostics are incorrect.  They should be [Z L T-2 ~> m2 s-2].

Reply:
Thanks. I have made the corrections.